### ISSUE TITLE
express upgrade command fails 

### ISSUE THREAD

**Issue by jb4free** [platform9/express-cli#39]
$ express upgrade
Traceback (most recent call last):
  File "/home/jb4free/.virtualenvs/express-cli/bin/express", line 11, in <module>
    load_entry_point('express-cli', 'console_scripts', 'express')()
  File "/home/jb4free/.virtualenvs/express-cli/local/lib/python2.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/jb4free/.virtualenvs/express-cli/local/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/jb4free/.virtualenvs/express-cli/local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/jb4free/.virtualenvs/express-cli/local/lib/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/jb4free/.virtualenvs/express-cli/local/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/jb4free/.virtualenvs/express-cli/local/lib/python2.7/site-packages/click/decorators.py", line 27, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/jb4free/code/express-cli/pf9/express.py", line 91, in upgrade
    click.echo(ver.get_latest_json())
AttributeError: Pf9ExpVersion instance has no attribute 'get_latest_json'

**Comment by mattroark** [comment]
I can verify the fix works.
```bash
ubuntu@pmkft-cli:~$ pf9ctl upgrade
{'version': 'v1.3.3', 'url_tar': 'https://api.github.com/repos/platform9/express/tarball/v1.3.3', 'url_zip': 'https://api.github.com/repos/platform9/express/zipball/v1.3.3'}
A newer version of Platform9 Express is avialable
Platform9 Express upgrade complete
```

**Comment by jb4free** [comment]
closed by #38
